deploy/crds/chaosengine_crd.yaml

```diff
@@ -171,12 +171,1828 @@ spec:
                                         type: string
                                   source:
                                     type: object
+                                    description: The source where we have to run the
+                                      command It will run in inline(inside experiment
+                                      itself) mode if source is nil
```

Comment:
```suggestion
                                    description: The external pod where we have to run the
                                      probe commands. It will run the commands inside the experiment pod itself(inline mode) if source contains a nil value
```

Reply:
This is not updated yet, please check